### PR TITLE
Changing follow user button in menu bar

### PR DESCRIPTION
**Pull request by rudy-patel**
### What is it?
This PR changes the icon for the follow user action in the menu bar. The _search_ icon was a bit misleading, I think this works better!
image

### Tasks

- [x] run unit tests
- [x] run UI tests

### CODE REVIEW

File: app/src/main/res/drawable/baseline_group_add_18.xml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  <path
+      android:fillColor="@android:color/white"
+      android:pathData="M9.92,11.03C8.59,10.37 7.09,10 5.5,10s-3.09,0.37 -4.42,1.03C0.41,11.36 0,12.07 0,12.81V14h11v-1.19C11,12.07 10.59,11.36 9.92,11.03z"/>
+  <path
+      android:fillColor="@android:color/white"
+      android:pathData="M12.92,11.03c-0.76,-0.38 -1.57,-0.64 -2.42,-0.82c0.92,0.52 1.5,1.52 1.5,2.6V14h2v-1.19C14,12.07 13.59,11.36 12.92,11.03z"/>
+  <path
+      android:fillColor="@android:color/white"
+      android:pathData="M16,7.5l0,-1.5l-1,0l0,1.5l-1.5,0l0,1l1.5,0l0,1.5l1,0l0,-1.5l1.5,0l0,-1z"/>
+</vector>
```
Comment (rudy-patel) [inline app/src/main/res/drawable/baseline_group_add_18.xml:22]: This is the new vector asset for the **follow user** icon, size `18`!

File: app/src/main/res/drawable/baseline_group_add_20.xml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  <path
+      android:fillColor="@android:color/white"
+      android:pathData="M15.03,12.37c-0.92,-0.54 -1.94,-0.92 -3.02,-1.14c0.92,0.64 1.48,1.71 1.48,2.86V16H16v-1.91C16,13.39 15.64,12.72 15.03,12.37z"/>
+  <path
+      android:fillColor="@android:color/white"
+      android:pathData="M10,10c1.66,0 3,-1.34 3,-3s-1.34,-3 -3,-3C9.79,4 9.58,4.02 9.38,4.06C10.07,4.85 10.5,5.87 10.5,7c0,1.13 -0.43,2.14 -1.12,2.93C9.58,9.98 9.79,10 10,10z"/>
+  <path
+      android:fillColor="@android:color/white"
+      android:pathData="M19.5,7.75l-1.75,0l0,-1.75l-1.5,0l0,1.75l-1.75,0l0,1.5l1.75,0l0,1.75l1.5,0l0,-1.75l1.75,0z"/>
+</vector>
```
Comment (rudy-patel) [inline app/src/main/res/drawable/baseline_group_add_20.xml:22]: This is the new vector asset for the **follow user** icon, size `20`!

File: app/src/main/res/drawable/baseline_group_add_24.xml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  <path
+      android:fillColor="@android:color/white"
+      android:pathData="M8,13c-2.67,0 -8,1.34 -8,4v3h16v-3C16,14.34 10.67,13 8,13z"/>
+  <path
+      android:fillColor="@android:color/white"
+      android:pathData="M12.51,4.05C13.43,5.11 14,6.49 14,8s-0.57,2.89 -1.49,3.95C14.47,11.7 16,10.04 16,8S14.47,4.3 12.51,4.05z"/>
+  <path
+      android:fillColor="@android:color/white"
+      android:pathData="M16.53,13.83C17.42,14.66 18,15.7 18,17v3h2v-3C20,15.55 18.41,14.49 16.53,13.83z"/>
+</vector>
```
Comment (rudy-patel) [inline app/src/main/res/drawable/baseline_group_add_24.xml:22]: This is the new vector asset for the **follow user** icon, size `24`!

File: app/src/main/res/menu/menu.xml [hunk 4]
```diff
@@ -3,12 +3,12 @@
...
-        android:title="User Search"
-        app:showAsAction="always" />
+        android:icon="@drawable/baseline_group_add_24"
+        android:title="@string/user_search"
+        app:showAsAction="ifRoom" />
     <item
         android:id="@+id/logout_button"
         android:icon="@drawable/ic_baseline_logout_24"
-        android:title="Logout"
+        android:title="@string/logout"
```
Comment (rudy-patel) [inline app/src/main/res/menu/menu.xml:12]: Extracting string resource...

File: app/src/main/res/menu/menu.xml [hunk 5]
```diff
@@ -3,12 +3,12 @@
     xmlns:app="http://schemas.android.com/apk/res-auto">
     <item
         android:id="@+id/follow_user_search_button"
-        android:icon="@android:drawable/ic_menu_search"
-        android:title="User Search"
-        app:showAsAction="always" />
+        android:icon="@drawable/baseline_group_add_24"
+        android:title="@string/user_search"
+        app:showAsAction="ifRoom" />
```
Comment (rudy-patel) [inline app/src/main/res/menu/menu.xml:8]: The change from `always` to `ifRoom` was an IntelliJ suggestion

**Review by jaskim9824 — APPROVED** [review]

File: app/src/main/res/drawable/baseline_group_add_18.xml (hunk 1 above)
Comment (JuliePilz) [reply app/src/main/res/drawable/baseline_group_add_18.xml:22]: Just a quick question, what are these changes .DS_store files?

**Review by kaden-dreger — APPROVED** [review]

**Review by JuliePilz — APPROVED** [review]
Looks good! My only question is waht those changes .DS_Store files are, but I'm assuming it's not a big deal.

File: app/src/main/res/drawable/baseline_group_add_18.xml (hunk 1 above)
Comment (rudy-patel) [reply app/src/main/res/drawable/baseline_group_add_18.xml:22]: `DS_STORE` files just help when navigating the file explorer on mac. They aren't really changes, they just sneak into my changeset as I'm working on stuff. Ideally, these files can all be excluded from git entirely (we can put them into our `.gitignore`). 

Nothing to worry about though really.

**Pull request merged**